backend/config.js

```diff
@@ -0,0 +1,11 @@
+const config = {
+  environment: process.env.NODE_ENV || 'dev',
```

Comment:
I would recommend changing `dev` to `development` for consistency because express uses the [same naming convention](https://github.com/expressjs/express/blob/master/lib/application.js#L71) within their package.

Reply:
done